libs/providers/config-cat-web/README.md

```diff
@@ -0,0 +1,148 @@
+# ConfigCat Web Provider
+
+This provider is an implementation for [ConfigCat](https://configcat.com) a managed feature flag service.
+
+## Installation
+
+```
+$ npm install @openfeature/config-cat-web-provider
+```
+
+#### Required peer dependencies
+
+The OpenFeature SDK is required as peer dependency.
+
+The minimum required version of `@openfeature/web-sdk` currently is `1.0.0`.
+
+The minimum required version of `configcat-js-ssr` currently is `7.1.2`.
+
+```
+$ npm install @openfeature/client-sdk configcat-js-ssr
+```
+
+## Usage
+
+The ConfigCat provider uses the [ConfigCat JavaScript (SSR) SDK](https://configcat.com/docs/sdk-reference/js-ssr/).
+
+It can either be created by passing the ConfigCat SDK options to ```ConfigCatProvider.create``` or
+the ```ConfigCatProvider``` constructor.
+The available options can be found in the [ConfigCat JavaScript (SSR) SDK](https://configcat.com/docs/sdk-reference/js-ssr/).
+
+The ConfigCat Web Provider only supports the `AutoPolling` mode because it caches all evaluation data to support synchronous evaluation of feature flags.
+
+### Example using the default configuration
+
+```javascript
+import { ConfigCatProvider } from '@openfeature/config-cat-web-provider';
+
+const provider = ConfigCatProvider.create('<sdk_key>');
+OpenFeature.setProvider(provider);
+```
+
+### Example using different polling options and a setupHook
+
+```javascript
+import { ConfigCatProvider } from '@openfeature/config-cat-web-provider';
+
+const provider = ConfigCatProvider.create('<sdk_key>', PollingMode.AutoPoll, {
+  setupHooks: (hooks) => hooks.on('clientReady', () => console.log('Client is ready!')),
+});
+
+OpenFeature.setProvider(provider);
+```
+
+## Evaluation Context
+
+ConfigCat only supports string values in its "evaluation
+context", [there known as User Object](https://configcat.com/docs/advanced/user-object/).
```

Comment:
This is [not true any more](https://configcat.com/docs/sdk-reference/js-ssr/#user-object-attribute-types) (since JS SSR SDK v8.0), so this section should be revised.